docs/build/linux-ubuntu.md

```diff
@@ -262,3 +257,31 @@ Once the image is built, it may be used to build parts of Keyman.
     keymanapp/keyman-linux-builder:latest \
     android/build.sh --debug
   ```
+
+### Customizing the builder
+
+You can use Docker [build args](https://docs.docker.com/build/guide/build-args/) to customize the image build. As an example, the following will build an image explicitly with Ubuntu 23.04 and Node.js 21
```

Comment:
I guess the sample uses that's Node 20 :smile: 

```suggestion
You can use Docker [build args](https://docs.docker.com/build/guide/build-args/) to customize the image build. As an example, the following will build an image explicitly with Ubuntu 23.04 and Node.js 20
```

Reply:
21 on the brain for unrelated reasons.